lib/vsc/install/ci.py

```diff
@@ -62,19 +65,68 @@
 PY3_ONLY = 'py3_only'
 PY3_TESTS_MUST_PASS = 'py3_tests_must_pass'
 RUN_SHELLCHECK = 'run_shellcheck'
+ENABLE_GITHUB_ACTIONS = 'enable_github_actions'
 
 logging.basicConfig(format="%(message)s", level=logging.INFO)
 
 
 def write_file(path, txt):
     """Write specified contents to specified path."""
+
+    if not os.path.exists(os.path.dirname(path)):
+        os.makedirs(os.path.dirname(path))
     try:
         with open(path, 'w') as handle:
             handle.write(txt)
         logging.info("Wrote %s", path)
     except (IOError, OSError) as err:
         raise IOError("Failed to write %s: %s" % (path, err))
 
+def gen_github_action():
+    """
+    Generate tox.ini configuration file for github actions.
+    """
+    logging.info('[%s]', GITHUB_ACTIONS)
+    vsc_ci_cfg = parse_vsc_ci_cfg()
+
+    name_url = "https://github.com/hpcugent/vsc-install"
```

Comment:
this won't work yet as I need to get this line dynamically for each repo, I tried the code of headers and it does not work. @boegel suggestions?

Reply:
I just noticed the headers script is used with `REPO_BASE_DIR=$PWD`, maybe this will require the same trick.

Reply:
well, found a way to fix it, but I have to admit I don't understand how it works.